### PR TITLE
Ebos cleanups

### PR DESCRIPTION
**Pull request by andlaus**
this PR contains a bunch of cleanups -- like typo and whitespace fixes, message improvements, etc -- for ebos. It also adds the possibility to disable aquifer handling in ebos, because I had some severe issues with them for a deck I was recently working on. none of them should have any effect on the simulation results of `flow`...

### CODE REVIEW

**Comment by andlaus** [comment]
jenkins build this please

File: CMakeLists.txt [hunk 1]
```diff
@@ -259,6 +259,14 @@ opm_add_test(ebos_altidx
...
   LIBRARIES "opmsimulators")
 
+opm_add_test(ebos_plain
+  ONLY_COMPILE
+  DEFAULT_ENABLE_IF ${EBOS_DEBUG_EXTENSIONS_DEFAULT_ENABLE_IF}
+  SOURCES ebos/ebos_plain.cc
+  EXE_NAME ebos_plain
+  DEPENDS "opmsimulators"
+  LIBRARIES "opmsimulators")
+
```
Comment (totto82) [inline CMakeLists.txt]: I assume this is a preparation for a ebos multiplexer binary since the ebos binary is kept.

**Comment by totto82** [comment]
Thanks for the clean-up.

File: CMakeLists.txt (hunk 1 above)
Comment (andlaus) [reply CMakeLists.txt]: Not really: I just noticed could only be build with the dune based BS, but not with the default one. (I exclusively use the former, except occasionally for testing.) Anyway, thanks for merging.

**Pull request merged**